test/test_datapipe.py

```diff
@@ -1,9 +1,11 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
 import io
+import itertools
 import unittest
 import warnings
 
 from collections import defaultdict
+from functools import partial
```

Comment:
Last nit: Could you please also remove `partial` here, as it's not used?

Reply:
Done! Not sure how linting missed that, sorry.